hell/adventcalendar/2024/23/index.md

```diff
@@ -2,51 +2,188 @@
 title: "The devil is in the &lt;details&gt;"
```

Comment:
@matuzo this renders fine on the page but on the browser title bar it display as text. Not sure if there's anything that can be done. If not, I'd advise in removing the `<...>` for readability? It's up to you.